workers/python/python/extraction/extraction.py

```diff
@@ -0,0 +1,27 @@
+from bullmq import Job
+
+from workers import create_worker
+from enums import ExtractionBackend
+from database import database
+from extraction.unstructured import unstructuredExtraction
+
+EXTRACTION_QUEUE_NAME = "files:extraction:python"
+
+
+async def processExtraction(job: Job, job_token):
+    data = job.data
+    file_id = data['fileId']
+
+    file = database.get_collection('file').find_one({"_id": file_id})
```

Comment:
synchronous operation in async function, should be handled in a separate thread

Reply:
Btw there is asyncio client for mongo with the same syntax - [motor](https://motor.readthedocs.io/en/stable/api-asyncio/asyncio_motor_client.html)